app/src/main/java/com/framgia/soundclound/util/Constant.java

```diff
@@ -12,6 +17,7 @@
     public static final String OFF_SET = "offset";
     public static final String LIMIT = "limit";
     public static final String ERROR_NULL = "null_data";
+    public static final String[] PERMISSON = new String[]{Manifest.permission.READ_EXTERNAL_STORAGE};
```

Comment:
ERROR: [MALICIOUS_CODE][MS_MUTABLE_ARRAY] com.framgia.soundclound.util.Constant.PERMISSON is a mutable array
http://findbugs.sourceforge.net/bugDescriptions.html#MS_MUTABLE_ARRAY